Sources/CorePayments/AnalyticsEventData.swift

```diff
@@ -41,7 +42,9 @@ struct AnalyticsEventData: Encodable {
 
     let clientOS: String = UIDevice.current.systemName + " " + UIDevice.current.systemVersion
 
-    let component = "ppcpmobilesdk"
+    let component = "ppcpclientsdk"
```

Comment:
😈 advocate - will changing the component name cause more trouble than it's worth?

The pro is to mirror BT analytics component name, but the con is definitely a pain of needing to manage data across components.

Reply:
That's a great question! I think if we plan to document this and use it for future SDKs, then yes. It's also probably easier to make the change now vs once we have a ton more data coming through the SDK. It also allows us to be consistent across products vs doing `mobile` in one and `client` in another, which I think has been the goal with moving to FPTI - consistency.

Reply:
> It also allows us to be consistent across products vs doing mobile in one and client in another, which I think has been the goal with moving to FPTI - consistency.

I agree with this, but also we don't know the analytics story for PPCP server or PPCP web. Since PPCP web is baked into smart payment buttons, there is low chance they'll do a rename to align with us. I just asked in our #pp-mobile-sdk-analytics Slack channel to see if anyone knows.

Reply:
I guess what I'm saying is I kind of doubt the PPCP web product will want to change their component name from whatever it is to `ppcpclientsdk` just to match ours, so is there really a point to changing ours?

Reply:
Yeh, the server SDKs won't use `<product>clientsdk`, so mobile might just always be it's own special name

Reply:
Ah - good point about server. We can revert it. I guess my 👿 advocate argument would be why did we change it on BT iOS. Lol. I don't feel strongly enough to argue in either direction besides it being divergent from what we decided on BT, so if you think we should use `mobile` I can make that change.

Reply:
Yeh, I get that. I think changing on BT made more sense since we have old events coming in that use a different name (from Arachne), so now they're consistent.

I could go either way too!